more_autograding_examples/image_diff_mirror/config/config.json

```diff
@@ -1,23 +1,13 @@
 {
     "autograding" : {
         "submission_to_runner" : [ "*.txt", "*.png" ],
-	"work_to_details" : ["*.png"]
+	       "work_to_details" : ["**/*.png"]
     },
     "max_submission_size" : 2000000,
     "testcases" : [    
-        // *************** COMPILATION *****************
-        {
-            "type" : "Compilation",
-            "title" : "Compilation",
-            //Note the multistep compilation which first 
-            //calls cmake and then make. 
-            "command" : ["g++ *.cpp"],
-            "executable_name" : "a.out",
-            "points" : 4
-        },    
         {
             "title" : "Mirror",
-            "command" : "./a.out",
+            "command" : "sleep .001",
```

Comment:
Can command be empty if all someone is interested in is the validation step?

Reply:
Good catch. I was under the impression that all testcases required a command (I believe that was a constraint at one point). Commands don't really make sense for this gradeable, as it just takes a user provided image and checks to see if it matches the instructor provided one, so I added the sleep as a workaround. As it turns out, we must have removed the need for a command at some point, so I was able to nix them for this gradeable in my most recent commit.